### PR TITLE
Make tests use dev database config

### PR DESCRIPTION
**Pull request by yguedidi**
Benefits:
- easier testing on different databases locally
- prepare for the use of environment variables for configuration

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -97,5 +97,37 @@ public function registerContainerConfiguration(LoaderInterface $loader)
...
+                $scheme = 'pgsql';
+                break;
+            case 'pdo_sqlite':
+                $scheme = 'sqlite';
+                break;
+            default:
+                throw new \RuntimeException('Unsupported database driver: ' . $container->getParameter('database_driver'));
+        }
+
+        $container->setParameter('database_scheme', $scheme);
```
Comment (yguedidi) [inline app/AppKernel.php:122]: needed because using directly the driver value as a scheme doesn't work

File: app/AppKernel.php [hunk 2]
```diff
@@ -97,5 +97,37 @@ public function registerContainerConfiguration(LoaderInterface $loader)
...
+
+        if ('sqlite' === $scheme) {
+            $container->setParameter('database_name', $container->getParameter('database_path'));
+        }
```
Comment (yguedidi) [inline app/AppKernel.php:126]: added in order to use only the database_name in the url

File: app/AppKernel.php [hunk 3]
```diff
@@ -97,5 +97,37 @@ public function registerContainerConfiguration(LoaderInterface $loader)
...
+
+        $container->setParameter('database_user', (string) $container->getParameter('database_user'));
+        $container->setParameter('database_password', (string) $container->getParameter('database_password'));
+        $container->setParameter('database_port', (string) $container->getParameter('database_port'));
+        $container->setParameter('database_socket', (string) $container->getParameter('database_socket'));
```
Comment (yguedidi) [inline app/AppKernel.php:131]: needed because it doesn't work to concatenate a null value in parameters

File: app/config/config.yml [hunk 4]
```diff
@@ -8,6 +8,7 @@ parameters:
     # Allows to use the live reload feature for changes in assets
     use_webpack_dev_server: false
     craue_config.cache_adapter.class: Craue\ConfigBundle\CacheAdapter\SymfonyCacheComponentAdapter
+    env(DATABASE_URL): '%database_scheme%://%database_user%:%database_password%@%database_host%:%database_port%/%database_name%?unix_socket=%database_socket%&charset=%database_charset%'
```
Comment (yguedidi) [inline app/config/config.yml:11]: where the magic happens, in a backward compatible way :)

**Review by j0k3r — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Pull request merged**